DM-49333: Add new metadata parameter

timj opened the pull request:
@czwa looking at the code it seems to me that we should be doing what display_firefly does and writing a temp FITS file and then asking ginga to load the FITS file? Did you try that and it didn't work?

Is there a test tutorial on data.lsst.cloud that I can use to see if things work?

timj committed:
Add new metadata parameter to _mtv

timj committed:
Simplify scons build

timj committed:
Fix some ds9 references

czwa commented:
I don't think there's a method to read from disk (I've only lightly poked at the display object already open in one of my notebooks).  I think both the astrowidgets intermediate and the base ginga code assume everything is already in memory (but it's been 2+ years since I last looked).

I've updated my "minimal astrowidgets example" notebook at the USDF (`/sdf/home/c/czw/dev/tools/minimal_aw_example.ipynb`) and would be happy to try and get this added as a test tutorial (so it doesn't bitrot the way it had up until today).

timj commented:
`ginga.util.io.io_fits.load_file` returns an `AstroImage` object.

I was testing it when I came across a small problem https://github.com/ejeschke/ginga/issues/1125

timj committed:
Use temp fits file to load ginga

This removes the need for the special WCS class and uses the
same FITS file writing as used in firefly and ds9 interfaces.

timj commented:
@czwa please can you review this and try it out? You will need the afw branch that matches. It works for me on data.lsst.cloud and I can remove all the WCS code from the package (and I think the WCS responsiveness might be better now).

timj committed:
Note that title and metadata might now be used

czwa reviewed (approved):
This appears to support all of the same behavior that the current version of display_astrowidgets, so I have no major concerns.
